pr-reminders/slack.py

```diff
@@ -34,16 +34,26 @@ def send_message(self, channel, message, header):
 
     def send_message_with_user_mention(self, channel, template: Template, user_email,
                                        header):
-        user_response = self.client.users_lookupByEmail(email=user_email)
-        user_id = user_response['user']['id']
-        message = template.substitute(user=f'<@{user_id}>')
-        return self.send_message(channel, message, header)
+        try:
+            user_response = self.client.users_lookupByEmail(email=user_email)
+            user_id = user_response['user']['id']
+            message = template.substitute(user=f'<@{user_id}>')
+            return self.send_message(channel, message, header)
+        except Exception as e:
+            logging.error(f"Unexpected error when looking up user by email: {str(e)}")
+            message = template.substitute(user=f'<@{user_email}>')
+            return self.send_message(channel, message, header)
 
     def send_message_with_user_mentions(self, channel, template: Template,
                                         user_emails: List[str], header):
-        user_responses = [self.client.users_lookupByEmail(
-            email=email) for email in user_emails]
-        user_ids = [response['user']['id'] for response in user_responses]
-        user_id_references = [f'<@{user_id}>' for user_id in user_ids]
+        user_id_references = []
+        for email in user_emails:
+            try:
+                user_response = self.client.users_lookupByEmail(email=email)
+                user_id = user_response['user']['id']
+                user_id_references.append(f'<@{user_id}>')
+            except Exception as e:
+                logging.error(f"Unexpected error looking up {email}: {str(e)}")
+                user_id_references.append(f'<@{email}>')
```

Comment:
I think it would be better to just do:
```
user_id_references.append(f'{email}')
```
as we cannot do any reference to a slack user if the person is not resolvable.